src/SIL.LCModel/LcmCache.cs

```diff
@@ -1434,7 +1434,7 @@ internal IFwMetaDataCache MetaDataCache
 		/// <summary>
 		/// Constructor.
 		/// </summary>
-		internal LcmCache()
+		public LcmCache()
```

Comment:
This can't be `public` - `LcmCache` isn't intended to be created directly, only through one of the `CreateCache...` methods.